testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.minItems_03/minItems.openapi.todo

```diff
@@ -0,0 +1,118 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+
+  "paths": {
+    "/request/jschema/rules/minItems": {
+      "get": {
+        "responses": {
+
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "type": "string",
+                    "example": "str"
+                  }
+                }
+              }
+            }
+          },
+
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "minItems": 2,
+                  "items": {
+                    "anyOf": [
+                      {"type": "string", "example": "str"},
+                      {"type":"integer", "example": 1}
+                    ]
+                  }
+                }
+              }
+            }
+          },
+
+          "202": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "minItems": 3,
+                  "items": {
+                    "anyOf": [
+                      {"type": "integer", "example": 1},
+                      {"type": "number", "example": 2.3},
+                      {"type": "string", "example": "abc"}
+                    ]
+                  }
+                }
+              }
+            }
+          },
+
+          "203": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "anyOf": [
+                      {
+                        "type": "integer",
+                        "example": 1
+                      },
+                      {
+                        "type": "integer",
+                        "example": 2
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          },
+
+          "204": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
```

Comment:
This array schema is wrong, does not accord to the JSight

```
  204
    [ // { type: "array", nullable: true, minItems: 0 }
    ]
```

Reply:
fixed https://github.com/jsightapi/jsight-api-core/pull/15/commits/2acbd932726b3e5514ef90160b7567b604cbc271